scripts/qa_crowbarsetup.sh

```diff
@@ -633,6 +633,14 @@ function onadmin_prepare_cloud_repos()
                 ;;
         esac
     fi
+
+    # Ensure Pool/Updates repos are always configured on admin server
+    for repo in Pool Updates; do
+        name="SUSE-Cloud-$(echo $cloudsource | sed "s/[^0-9]//g")-$repo"
```

Comment:
Ah, yes, it's relevant and I guess wouldn't work with M4 :/

How can I extract the SUSE Cloud version from cloudsource?